src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_diffedit.py

```diff
@@ -1336,7 +1336,7 @@ def __call__(
         self,
         prompt: Optional[Union[str, List[str]]] = None,
         mask_image: Union[torch.FloatTensor, PIL.Image.Image] = None,
-        image_latents: torch.FloatTensor = None,
+        image_latents: Union[torch.FloatTensor, PIL.Image.Image] = None,
```

Comment:
`image_latents` is not a good name when we're accepting PIL images too. `image` works for me.

Reply:
Agree, happy to deprecate the name via `**kwargs`